sysml.library/Systems Library/Items.sysml

```diff
@@ -67,9 +97,16 @@ package Items {
 		 * Constraints that have been checked by this item.
 		 */
 		abstract ref constraint checkedConstraints: ConstraintCheck[0..*] :> constraintChecks, enactedPerformances;
-
 	}
 	
+	/**
+	 * Touching occurrences are just outside each other and happen at the same time.
+	 */
```

Comment:
"Touching occurrences" should be "Touching items".

Reply:
K, will fix along with previous comment (and commit after a bit to let more feedback come in).

Reply:
Please make any updates to branch ST6RI-540 now. I don't want to hold up the PR any longer.